File: src/helpers/token.js
const jwt = require('jsonwebtoken');

const generateToken = (email, guid) => {
    const token = jwt.sign({
        email,
        guid,
    }, process.env.JWT_ACCESS_TOKEN, {
        expiresIn: "8h",
    });
    return token;
}
module.exports = generateToken

File: src/helpers/token.py
import jwt
import datetime
from config import Config

def generate_token(email, user_id):
    token = jwt.encode({
        'email': email, 
        'id': user_id, 
        # 'exp': datetime.datetime.utcnow() + datetime.timedelta(hours=1)}, 
        },
        Config.SECRET_KEY)
    return token
